stores/metrics.go

```diff
@@ -226,6 +226,10 @@ func (s *SQLStore) PerformanceMetrics(ctx context.Context, start time.Time, n ui
 }
 
 func (s *SQLStore) RecordContractMetric(ctx context.Context, metrics ...api.ContractMetric) error {
+	if len(metrics) == 0 {
```

Comment:
If you add that here we might not realise that something is calling this endpoint with no metrics and it would be better not to send that http request in the first place.